eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/TcpClient.java

```diff
@@ -75,8 +75,7 @@ public abstract class TcpClient implements Closeable {
     private transient ScheduledFuture<?> heartTask;
 
     protected static final ScheduledThreadPoolExecutor scheduler = new ScheduledThreadPoolExecutor(
```

Comment:
Suggest add createScheduledThreadPoolExecutor in Threadfactory Class.

Reply:
I will fix those problems and resubmit this pr later